doc/manual/expressions/builtins.xml

```diff
@@ -280,8 +280,15 @@ with import (fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/nixo
 
 stdenv.mkDerivation { … }
 </programlisting>
+    <para>The fetched tarball is cached for a certain amount of time
```

Comment:
Oops, one more thing: there also needs to be a `</para>` before this opening `<para>`.